tests/common/dualtor/dual_tor_utils.py

```diff
@@ -1072,7 +1072,7 @@ def check_nexthops_single_uplink(portchannel_ports, port_packet_count, expect_pa
 def check_nexthops_single_downlink(rand_selected_dut, ptfadapter, dst_server_addr,
                                    tbinfo, downlink_ints):
     HASH_KEYS = ["src-port", "dst-port", "src-ip"]
-    expect_packet_num = 10000
+    expect_packet_num = 1000
```

Comment:
could you also add a tolerance for the packet number? we used to have this in the old test and since the changes to this test we're failing because we don't have these tolerances anymore. I was working on a fix to add these, but if you don't mind incorporating in your PR to save some time, this is the code I had:

right after line 1075:
```
+    pkt_num_lo = expect_packet_num * (1.0 - 0.25)
+    pkt_num_hi = expect_packet_num * (1.0 + 0.25)
```

on line 1101:
```
-        if count > 0 and count != expect_packet_num:
+        if count > 0 and (count > pkt_num_hi or count < pkt_num_lo):
```

Reply:
I think this tolerance was for the balance check, do we still need this since we don't expect balance in check_nexthops_single_downlink()?